tasks/db_deploy/install/orca_reconcile_sql.py

```diff
@@ -0,0 +1,305 @@
+    """
+    Full SQL for creating the reconcile_phantom_report table. 
+
+    Returns:
+        (sqlalchemy.sql.element.TextClause): SQL for creating reconcile_phantom_report table.
+    """
+    return text(
+        """
+            -- Create reconcile_phantom_report table
+            CREATE TABLE IF NOT EXISTS reconcile_phantom_report  
+            ( 
+              job_id          	int8 NOT NULL
+            , collection_id   	text NOT NULL
+            , granule_id      	text NOT NULL
+            , filename        	text NOT NULL
+            , key_path        	text NOT NULL
+            , orca_etag       	text NOT NULL
+            , orca_last_update	timestamp with time zone NOT NULL
+            , orca_size       	int8 NOT NULL
+            , CONSTRAINT PK_reconcile_catalog_mismatch_report PRIMARY KEY(job_id,collection_id,granule_id,key_path)
+            , CONSTRAINT FK_reconcile_job_phantom_report FOREIGN KEY(job_id) REFERENCES reconcile_job(id)
+            );
+            -- Comment
+            COMMENT ON TABLE reconcile_phantom_report IS 'Table that identifies objects that exist in the ORCA catalog and do not exist in the ORCA S3 bucket.';
+            COMMENT ON COLUMN reconcile_phantom_report.job_id IS 'Job the mismatch or missing granule was found in. References the reconcile_job table.';
+            COMMENT ON COLUMN reconcile_phantom_report.collection_id IS 'Cumulus Collection ID value from the ORCA catalog.';
+            COMMENT ON COLUMN reconcile_phantom_report.granule_id IS 'Cumulus granuleID value from the ORCA catalog.';
+            COMMENT ON COLUMN reconcile_phantom_report.filename IS 'Filename of the object from the ORCA catalog.';
+            COMMENT ON COLUMN reconcile_phantom_report.key_path IS 'key path and filename of the object in the ORCA catalog.';
+            COMMENT ON COLUMN reconcile_phantom_report.orca_etag IS 'etag of the object as reported in the ORCA catalog.';
+            COMMENT ON COLUMN reconcile_phantom_report.orca_last_update IS 'Last update of the object as reported in the ORCA catalog.';
+            COMMENT ON COLUMN reconcile_phantom_report.orca_size IS 'Size in bytes of the object as reported in the ORCA catalog.';
+        """
+    )
+
+#TBD discuss with team on SQL code for list of buckets
+def orca_archive_location_bucket() -> TextClause:
+    """
+    Full SQL for creating the orca_archive_location_bucket table. 
+
+    Returns:
+        (sqlalchemy.sql.element.TextClause): SQL for creating orca_archive_location_bucket table.
+    """
+
+    return text(
+        """
+            -- Create orca_archive_location_:bucket_name
+            CREATE TABLE {orca_archive_location_:bucket_name}
+            (
+            PARTITION OF reconcile_s3_object
+            FOR VALUES IN (:bucket_name)
+            );
+            -- Comment
+            COMMENT ON TABLE orca_archive_location_:bucket_name IS 'Table for partition of S3 bucket.';
+        """
+    )
```

Comment:
not sure if this is correct.

Reply:
Per our meeting, no default partition will be created in order to surface the failure faster.